templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_oeb_header.html.twig

```diff
@@ -38,6 +39,7 @@
                                         class="c-actionmenu__menuitem whitespace-normal"
                                         data-actionmenu-menuitem
                                         data-actionmenu-current
+                                        :data-cy="`baseHeader:{{ project.name | wysiwyg }}`"
```

Comment:
see above

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/3206/commits/7b4b0bf5699fbdc089c67e5d24699f7ee264100f